packages/pds/src/mailer/index.ts

```diff
@@ -40,6 +41,13 @@ export class ServerMailer {
     })
   }
 
+  async sendAccountDelete(params: { token: string }, mailOpts: Mail.Options) {
```

Comment:
Random thought coming to mind— we could send the user their repo over email as some part of the process.

Reply:
yeah that'd be really nifty 👍 

maybe we get that in the work push around account portability